app/views/charge.njk

```diff
@@ -18,22 +18,22 @@
 
 {% set shouldShowEmail = (gatewayAccount.emailCollectionMode === 'MANDATORY') or (gatewayAccount.emailCollectionMode === 'OPTIONAL') %}
 
-{% macro paymentSummary() %}
+{% macro paymentSummary(main = true) %}
   <div class="payment-summary-wrap">
     <div class="payment-summary">
         <h2 class="govuk-heading-m">{{ __p("paymentSummary.title") }}</h2>
-        <p id="payment-description" class="govuk-body">
+        <p id="{{ 'payment-description' if main else 'secondary-payment-description' }}" class="govuk-body">
```

Comment:
Instead of using the word secondary - can we use `wallet` instead?
E.g. `wallet-payment-description`
Please let me know what you think?